app/views/galleries/show.rb

```diff
@@ -11,6 +11,12 @@ def page_title
       mustache[:page_title] ||= [@gallery.title, site_title].join(' - ')
     end
 
+    def head_links
+      mustache[:head_links] ||= begin
+        { items: [{ rel: 'alternate', type: 'application/rss+xml', href: galleries_url(format: 'rss') }] + super[:items] }
```

Comment:
is there anywhere we're populating anything other than :items?
I've copied this from app/views/portal/show.rb

The only other place the head_links method is being declared as far as I can see is in aplication_view.rb where it always only has :items.